src/test/org/codehaus/groovy/runtime/URLGetBytesTest.groovy

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright 2003-2015 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.codehaus.groovy.runtime
+
+/**
+ * Tests for {@link DefaultGroovyMethods} URL.getBytes() methods.
+ *
+ */
+class URLGetBytesTest extends GroovyTestCase {
+    void testGetBytesFromURLWithParameters() {
+        def url = new URL('http','groovy.codehaus.org',80, '/', new URLStreamHandler() {
```

Comment:
I know it's a dummy URLConnection but I'd probably change groovy.codehaus.org to groovy-lang.org.